Translations update from Hosted Weblate

weblate opened the pull request:
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [cetmix-tower-14.0-dev/cetmix_tower_server](https://hosted.weblate.org/projects/tower-server-14-0-dev/cetmix_tower_server/).


It also includes following components:

* [cetmix-tower-14.0-dev/cetmix_tower_yaml](https://hosted.weblate.org/projects/tower-server-14-0-dev/cetmix_tower_yaml/)

* [cetmix-tower-14.0-dev/cetmix_tower_git](https://hosted.weblate.org/projects/tower-server-14-0-dev/cetmix_tower_git/)



Current translation status:

![Weblate translation status](https://hosted.weblate.org/widget/tower-server-14-0-dev/cetmix_tower_server/horizontal-auto.svg)

<!-- This is an auto-generated comment: release notes by coderabbit.ai -->
## Summary by CodeRabbit

- **New Features**
	- Introduced Finnish language support with localized user interface text, error messages, and helper instructions to enhance the experience for Finnish-speaking users.
	- Added Croatian language support with a framework for future translations, facilitating localization for Croatian-speaking users.
	- Established Croatian language support for the `cetmix_tower_yaml` module, providing initial translation entries for user actions and error messages related to YAML data handling.
- **Bug Fixes**
	- Updated translation string formatting in the German language file for improved clarity.
<!-- end of auto-generated comment: release notes by coderabbit.ai -->

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/cetmix/cetmix-tower?pullRequest=210) <br/>All committers have signed the CLA.

badbole committed:
Added translation using Weblate (Croatian)

Translated using Weblate (Croatian)

Currently translated at 90.5% (460 of 508 strings)

Translated using Weblate (Croatian)

Currently translated at 39.7% (202 of 508 strings)

Translated using Weblate (Croatian)

Currently translated at 9.4% (9 of 95 strings)

Added translation using Weblate (Croatian)

Added translation using Weblate (Croatian)

Co-authored-by: Bole <bole@dajmi5.com>
Translate-URL: https://hosted.weblate.org/projects/tower-server-14-0-dev/cetmix_tower_git/hr/
Translate-URL: https://hosted.weblate.org/projects/tower-server-14-0-dev/cetmix_tower_server/hr/
Translation: cetmix-tower-14.0-dev/cetmix_tower_git
Translation: cetmix-tower-14.0-dev/cetmix_tower_server

Commit:
Translated using Weblate (German)

Currently translated at 4.1% (21 of 508 strings)

Co-authored-by: CetmixWeblateBot <weblate@cetmix.com>
Translate-URL: https://hosted.weblate.org/projects/tower-server-14-0-dev/cetmix_tower_server/de/
Translation: cetmix-tower-14.0-dev/cetmix_tower_server

jusle committed:
Added translation using Weblate (Finnish)

Added translation using Weblate (Finnish)

Co-authored-by: Jussi Lehto <jussi.lehto@cetmix.com>

ivs-cetmix reviewed (approved):
/ocabot merge nobump

CetmixGitDrone commented:
Hey, thanks for contributing! Proceeding to merge this for you.
Prepared branch [14.0-dev-ocabot-merge-pr-210-by-ivs-cetmix-bump-nobump](https://github.com/cetmix/cetmix-tower/commits/14.0-dev-ocabot-merge-pr-210-by-ivs-cetmix-bump-nobump), awaiting test results.

CetmixGitDrone commented:
Congratulations, your PR was merged at 9bcfca10e694f0feec868d33bca0858ac7c6c493. Thanks a lot for contributing to cetmix. ❤️
